service/awsconfig/drainer_framework.go

```diff
@@ -146,13 +147,30 @@ func newDrainerResourceRouter(config DrainerFrameworkConfig) (*framework.Resourc
 		}
 	}
 
+	var v8ResourceSet *framework.ResourceSet
+	{
+		c := v8.DrainerResourceSetConfig{
+			GuestAWSClients: awsClients,
+			Logger:          config.Logger,
+
+			GuestUpdateEnabled: config.GuestUpdateEnabled,
+			ProjectName:        config.ProjectName,
+		}
+
+		v8ResourceSet, err = v8.NewDrainerResourceSet(c)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+	}
```

Comment:
Yeah this is odd we have to do it and that it is like separated. This is only to get a different resync period for certain resources. I did not come up with a cool idea to improve this.

Reply:
I find it ok. Those are separate sets, just happen to reconcile on the same object.